chapters/drawing.txt

```diff
@@ -710,6 +717,21 @@ The vertices comprising a patch have no implied geometry, and are used as
 inputs to tessellation shaders and the fixed-function tessellator to
 generate new point, line, or triangle primitives.
 
+ifdef::VK_MESA_rect_list[]
+[[drawing-rect-lists]]
+=== Rectangle Lists
+
+When the primitive topology is ename:VK_PRIMITIVE_TOPOLOGY_RECT_LIST_MESA,
+each consecutive set of three vertices defines a single rectangle primitive,
+according to the equation:
+
+  {empty}:: [eq]#p~i~ = {v~3i~, v~3i+1~, v~3i+2~}#
+
+The number of primitives generated is equal to
+[eq]#{lfloor}pname:vertexCount/3{rfloor}#.
+
+They cannot be clipped, must be axis aligned, and cannot have depth gradient.
```

Comment:
These probably want to be RFC keywords (i.e. must:, cannot: etc). Also doesn't define provoking vertex behaviour